src/MPU6050_tockn.cpp

```diff
@@ -98,48 +122,195 @@ void MPU6050::calcGyroOffsets(bool console){
+* interrupts where the delta t between the interrupts is precisely known.
+*
+* @param interval Delta t between timer interrupts (in sec)
+*/
+void MPU6050::interruptUpdate (unsigned long interval)
+{
+	
+}
+
+/** Set full-scale gyro range.
+ * 0 = +/- 250 degrees/sec
+ * 1 = +/- 500 degrees/sec
+ * 2 = +/- 1000 degrees/sec
+ * 3 = +/- 2000 degrees/sec
+ *
+ * @param range New gyro sensitivity setting
+*/
+void MPU6050::setGyroSensitivity(uint8_t range)
+{
+	switch (range)
+	{
+		case 0:
+			gyroSensitivity = MPU6050_250DS_GYRO_SENSITIVITY;
+			break;
+			
+		case 1:
+			gyroSensitivity = MPU6050_500DS_GYRO_SENSITIVITY;
+			break;
+			
+		case 2:
+			gyroSensitivity = MPU6050_1000DS_GYRO_SENSITIVITY;
+			break;
+			
+		case 3:
+			gyroSensitivity = MPU6050_2000DS_GYRO_SENSITIVITY;
+			break;
+			
+		default:
+			return;
+	}
+	
+	writeBits(MPU6050_GYRO_CONFIG, MPU6050_GYRO_CONFIG_FS_SEL_BIT, MPU6050_GYRO_CONFIG_FS_SEL_LENGTH, range);
+}
+
+/** Set full-scale accel range.
+ * 0 = +/- 2g
+ * 1 = +/- 4g
+ * 2 = +/- 8g
+ * 3 = +/- 16g
+ *
+ * @param range New accel sensitivity setting
+ */
+void MPU6050::setAccelSensitivity(uint8_t range)
+{
+	switch (range)
+	{
+		case 0:
+			accelSensitivity = MPU6050_2G_ACCEL_SENSITIVITY;
+			break;
+			
+		case 1:
+			accelSensitivity = MPU6050_4G_ACCEL_SENSITIVITY;
+			break;
+			
+		case 2:
+			accelSensitivity = MPU6050_8G_ACCEL_SENSITIVITY;
+			break;
+			
+		case 3:
+			accelSensitivity = MPU6050_16G_ACCEL_SENSITIVITY;
+			break;
+		
+		default:
+			return;
+	}
+	
+	writeBits(MPU6050_ACCEL_CONFIG, MPU6050_ACCEL_CONFIG_FS_SEL_BIT, MPU6050_ACCEL_CONFIG_FS_SEL_LENGTH, range);
+}
+
+int16_t MPU6050::getRawAccelX(void)
+{
+	return read2BytesMPU6050(0x3B);
+}
+
+int16_t MPU6050::getRawAccelY(void)
+{
+	return read2BytesMPU6050(0x3D);
+}
+
+int16_t MPU6050::getRawAccelZ(void)
+{
+	return read2BytesMPU6050(0x3F);
+}
+
+int16_t MPU6050::getRawTemp(void)
+{
+	return read2BytesMPU6050(0x41);
+}
+
+int16_t MPU6050::getRawGyroX(void)
+{
+	return read2BytesMPU6050(0x43);
+}
+
+int16_t MPU6050::getRawGyroY(void)
+{
+	return read2BytesMPU6050(0x45);
+}
+
+int16_t MPU6050::getRawGyroZ(void)
+{
+	return read2BytesMPU6050(0x47);
+}
+
+int16_t MPU6050::getAccelX(void)
+{
+	return getRawAccelX() / accelSensitivity;
+}
+
+int16_t MPU6050::getAccelY(void)
+{
+	return getRawAccelY() / accelSensitivity;
+}
+
+int16_t MPU6050::getAccelZ(void)
+{
+	return getRawAccelZ() / accelSensitivity;
+}
+
+int16_t MPU6050::getGyroX(void)
+{
+	return (getRawGyroX() / gyroSensitivity) - gyroXoffset;
+}
+
+int16_t MPU6050::getGyroY(void)
+{
+	return (getRawGyroY() / gyroSensitivity) - gyroYoffset;
+}
+
+int16_t MPU6050::getGyroZ(void)
+{
+	return (getRawGyroZ() / gyroSensitivity) - gyroZoffset;
+}
+
+int16_t MPU6050::getTemp(void)
+{
+	return (getRawTemp() + 12412.0f) / 340.0f;
+}
+
+double MPU6050::getAccelAngleX(double accelX, double accelY, double accelZ)
+{
+	return atan2(accelY, accelZ + abs(accelX)) * 180 / PI;
```

Comment:
`getAccelAngle` ⇨ `getAccAngle`
Please don't change method name.  
It will lost backward compatibility.